### ISSUE TITLE
Run Windows on Travis CI?

### ISSUE THREAD

**Issue by cclauss** [douglaskastle/blender-addon-tester#3]
A comment in __.travis.yml__ says that Python can not be run on Travis CI Windows.  See: https://docs.travis-ci.com/user/languages/python/#running-python-tests-on-multiple-operating-systems  The command [__choco install blender python__](https://chocolatey.org/packages/blender) might just work.

**Comment by myselfhimself** [comment]
This looks solved does it not?

**Comment by douglaskastle** [comment]
No, this is for Windows on Travis, not github actions. Need to create a branch and show it failing

https://travis-ci.community/t/blender-crashes-under-travisci-windows-vm/5824
